qiita_pet/templates/study_base.html

```diff
@@ -152,12 +152,7 @@
     else if(confirm("Are you sure you want to delete {% raw study_title_msg %}?")) {
       $.post('{% raw qiita_config.portal_dir %}/study/delete/', { study_id: {{study_info['study_id']}} })
         .done(function ( data ) {
-          if(data.status == "error") {
```

Comment:
If there's an error with the deletion, the user will just see the page reload, would it reload to the same location? Meaning that the user wouldn't see any change?

Reply:
Excellent question! Well, the errors are now stored in a redis key as this is now gonna run as a job in the queue so the user will see the error on the reload.